app/exporter.py

```diff
@@ -35,7 +35,7 @@
 from prometheus_client.core import GaugeMetricFamily, REGISTRY
 
 DEFAULT_PORT = 9327
-DEFAULT_LOG_LEVEL = 'info'
+DEFAULT_LOG_LEVEL = 'debug'
```

Comment:
let's make this an env var like

```
DEFAULT_LOG_LEVEL = os.environ.get('S3_EXPORTER_LOG_LEVEL', 'info')
```

i believe we don't need tons of debug logs in prod

Reply:
let's return it back to info, it can be taken from config